### PR TITLE
fix(parseQueryString): do not remove '+' from query

### LINKED ISSUE #30: Pluses get replaced with spaces
**I'm submitting a bug report**

* **Library Version:**
1.1.3

**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
8.9.4

* **NPM Version:**
6.9.0

* **JSPM OR Webpack AND Version**
webpack 4.30.0

* **Browser:**
all

* **Language:**
all

**Current behavior:**
Passing `+` in a url parameter results in an argument to `activate` function where `+` is replaced with `[space]`

**Expected/desired behavior:**

* **What is the expected behavior?**
`+` is passed as is

* **What is the motivation / use case for changing the behavior?**
Previously, a path would be passed unencoded which correctly translated to view parameters. Now, when the `+` appears in the path, the line https://github.com/aurelia/path/blob/0be13f7710ddc81def93edaafd6ad3d339077c1d/src/index.js#L229 replaces it with a space

### PR DESCRIPTION
**Pull request by MaximBalaganskiy**
fixes #30

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ export function parseQueryString(queryString: string): Object {
     query = query.substr(1);
   }
 
-  let pairs = query.replace(/\+/g, ' ').split('&');
+  let pairs = query.split('&');
```
Comment (davismj) [inline src/index.js]: This wouldn't yield the desired behavior. + literals are spaces.

File: test/path.spec.js [hunk 2]
```diff
@@ -272,5 +272,7 @@ describe('query strings', () => {
     expect(parse('a[b][c][d]=e')).toEqual({a: {b: {c: {d: 'e'}}}});
     expect(parse('a[b][]=c&a[b][]=d&a[b][2][]=f&a[b][2][]=g')).toEqual({a: {b: ['c','d', ['f', 'g']]}});
     expect(parse('a[0]=b')).toEqual({a: ['b']});
+
+    expect(parse('a=b+c')).toEqual({a: 'b+c'});
```
Comment (davismj) [inline test/path.spec.js]:
```
expect(parse('a=b+c')).toEqual({a: 'b c'});
expect(parse('a=b%2Bc')).toEqual({a: 'b+c'});
```

I would pull if these tests pass. Fair?

Comment (MaximBalaganskiy) [reply test/path.spec.js]: The first line is exactly what I was trying to fix. If I pass %2B in a parameter it gets decoded in aurelia-browser-history and arrives as a '+', which is then replaced

Comment (MaximBalaganskiy) [reply test/path.spec.js]: I'm happy to close the PR if aurelia-browser-history gets fixed. As of now, this is a pretty serious bug which prevents using Asp.net Identity completely

**Comment by MaximBalaganskiy** [comment]
Closing as not working as the standard

**Pull request closed without merging**